spec/models/spree/inventory_unit_spec.rb

```diff
@@ -6,47 +6,6 @@
   let!(:stock_location) { create(:stock_location_with_items) }
   let(:stock_item) { Spree::StockItem.order(:id).first }
 
-  context "#backordered_for_stock_item" do
-    let(:order) { create(:order) }
-
-    let(:shipment) do
-      shipment = Spree::Shipment.new
-      shipment.shipping_methods << create(:shipping_method)
-      shipment.order = order
-      # We don't care about this in this test
-      allow(shipment).to receive(:ensure_correct_adjustment)
-      shipment.tap(&:save!)
-    end
-
-    let!(:unit) do
-      unit = shipment.inventory_units.build
-      unit.state = 'backordered'
-      unit.variant_id = stock_item.variant.id
-      unit.tap(&:save!)
-    end
-
-    # Regression for Spree #3066
-    it "returns modifiable objects" do
-      units = Spree::InventoryUnit.backordered_for_stock_item(stock_item)
-      expect { units.first.save! }.not_to raise_error
-    end
-
-    it "finds inventory units from its stock location " \
-       "when the unit's variant matches the stock item's variant" do
-      expect(Spree::InventoryUnit.backordered_for_stock_item(stock_item)).to eq [unit]
-    end
-
-    it "does not find inventory units that don't match the stock item's variant" do
-      other_variant_unit = shipment.inventory_units.build
-      other_variant_unit.state = 'backordered'
-      other_variant_unit.variant = create(:variant)
-      other_variant_unit.save!
-
-      expect(Spree::InventoryUnit.backordered_for_stock_item(stock_item))
-        .not_to include(other_variant_unit)
-    end
```

Comment:
Is this test still relevant for the method that was being wrapped (`backordered_per_variant`)?

Reply:
Thanks for checking. But I don't think that the spec is important now. I'm not even sure that we need InventoryUnit. It's a model designed to track the state of backorders, I think. I don't really want to get into that now though... :wink: